opentelemetry-sdk/src/opentelemetry/sdk/util/instrumentation.py

```diff
@@ -82,42 +85,56 @@ class InstrumentationScope:
     properties.
     """
 
-    __slots__ = ("_name", "_version", "_schema_url")
+    __slots__ = ("_name", "_version", "_schema_url", "_attributes")
 
     def __init__(
         self,
         name: str,
         version: Optional[str] = None,
         schema_url: Optional[str] = None,
+        attributes: Optional[Attributes] = None,
     ) -> None:
         self._name = name
         self._version = version
         if schema_url is None:
             schema_url = ""
         self._schema_url = schema_url
+        self._attributes = BoundedAttributes(attributes=attributes)
```

Comment:
not sure what the spec says but should there actually be any attribute length limits like for span or log attributes (see `SpanLimits`, `LogLimits`)?

Reply:
@mariojonke spec wording for this is very similar to the concept of SDK Resource, but there's no clarification if it should be exempt of limits as well. I presume it should be only immutable with no limits in the same way we have in SDK resources